package/yast2-network.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Mon Feb 22 11:52:09 UTC 2016 - mfilka@suse.com
+
+- bnc#960703
+  - sometimes wicked was started instead of NetworkManager in case
+    of AY installation. Moreover it could happen that both services
+    were "half-started" and network setup got broken.
+- 3.1.143
```

Comment:
The changelog is missing the Infiniband change from commit 648260d00cb3e02cc100bba986c5824d1d75f9c3